build/pumascript.js

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+
+var program = require('commander'),
+    puma = require('../dist/pumascript.js'),
+    fs  = require('fs');
+
+program
+.usage('<output> <filePaths ...>')
+.option('-o, --output <output>','Name and path of the output file')
+.parse(process.argv);
+
+//can be major than two coz the first args are the path for the output
+if (!program.args.length) {
+    program.help();
+}
+else{
+  if(program.output){
+    var pumaFile = '';
+
+    program.args.forEach(function(file){
+      pumaFile += fs.readFileSync(file,'utf8');
+    });
+
+    var parsedFile = puma.evalPuma(pumaFile);
+    console.log('PumaScript run Successfuly');
```

Comment:
Do we need this info? I think is irelevant if exit code is 0.  Something interesting is to show the progress when iterating the file list :

"Processing file <filename>" 

This will give information to the user in case the command fails at some point.